Sources/Core/Presenter/Channel/ChannelPresenter.swift

```diff
@@ -23,6 +23,8 @@ public final class ChannelPresenter: Presenter {
     public typealias FileAttachmentExtraDataCallback = (URL, Channel) -> Codable?
     /// A callback type for the adding an extra data for an image attachment.
     public typealias ImageAttachmentExtraDataCallback = (URL?, UIImage?, _ isVideo: Bool, Channel) -> Codable?
+    /// A callback type for preparing the message before sending.
+    public typealias MessageTextPreparationCallback = (Message) -> Message?
```

Comment:
Can you name this type `MessagePreparationCallback`?

Reply:
Sure!

Reply:
Done. The first approach I implemented was just changing the text, and I forgot to update the naming.